certify-service/src/test/resources/schema.sql

```diff
@@ -0,0 +1,111 @@
+    auth_factor character varying(25) NOT NULL,
+	psu_token character varying(256) NOT NULL,
+	public_key character varying NOT NULL,
+	expire_dtimes timestamp NOT NULL,
+	wallet_binding_id character varying(256) NOT NULL,
+	public_key_hash character varying(100) NOT NULL,
+	certificate character varying NOT NULL,
+	cr_dtimes timestamp NOT NULL,
+	CONSTRAINT pk_public_key_registry PRIMARY KEY (id_hash, auth_factor)
+);
+
+
+CREATE TABLE IF NOT EXISTS key_alias(
+    id character varying(36) NOT NULL,
+    app_id character varying(36) NOT NULL,
+    ref_id character varying(128),
+    key_gen_dtimes timestamp,
+    key_expire_dtimes timestamp,
+    status_code character varying(36),
+    lang_code character varying(3),
+    cr_by character varying(256) NOT NULL,
+    cr_dtimes timestamp NOT NULL,
+    upd_by character varying(256),
+    upd_dtimes timestamp,
+    is_deleted boolean DEFAULT FALSE,
+    del_dtimes timestamp,
+    cert_thumbprint character varying(100),
+    uni_ident character varying(50),
+    CONSTRAINT pk_keymals_id PRIMARY KEY (id),
+    CONSTRAINT uni_ident_const UNIQUE (uni_ident)
+);
+
+CREATE TABLE  IF NOT EXISTS key_policy_def(
+    app_id character varying(36) NOT NULL,
+    key_validity_duration smallint,
+    is_active boolean NOT NULL,
+    pre_expire_days smallint,
+    access_allowed character varying(1024),
+    cr_by character varying(256) NOT NULL,
+    cr_dtimes timestamp NOT NULL,
+    upd_by character varying(256),
+    upd_dtimes timestamp,
+    is_deleted boolean DEFAULT FALSE,
+    del_dtimes timestamp,
+    CONSTRAINT pk_keypdef_id PRIMARY KEY (app_id)
+);
+
+CREATE TABLE  IF NOT EXISTS key_store(
+	id character varying(36) NOT NULL,
+	master_key character varying(36) NOT NULL,
+	private_key character varying(2500) NOT NULL,
+	certificate_data character varying NOT NULL,
+	cr_by character varying(256) NOT NULL,
+	cr_dtimes timestamp NOT NULL,
+	upd_by character varying(256),
+	upd_dtimes timestamp,
+	is_deleted boolean DEFAULT FALSE,
+	del_dtimes timestamp,
+	CONSTRAINT pk_keystr_id PRIMARY KEY (id)
+);
+
+CREATE TABLE  IF NOT EXISTS consent_detail (
+    id UUID NOT NULL,
+    client_id VARCHAR NOT NULL,
+    psu_token VARCHAR NOT NULL,
+    claims VARCHAR NOT NULL,
+    authorization_scopes VARCHAR NOT NULL,
+    cr_dtimes TIMESTAMP DEFAULT NOW() NOT NULL,
+    expire_dtimes TIMESTAMP,
+    signature VARCHAR,
+    hash VARCHAR,
+    accepted_claims VARCHAR,
+    permitted_scopes VARCHAR,
+    PRIMARY KEY (id),
+    CONSTRAINT unique_client_token UNIQUE (client_id, psu_token)
+);
+
+CREATE TABLE  IF NOT EXISTS consent_history (
+    id UUID NOT NULL,
+    client_id VARCHAR NOT NULL,
+    psu_token VARCHAR NOT NULL,
+    claims VARCHAR NOT NULL,
+    authorization_scopes VARCHAR NOT NULL,
+    cr_dtimes TIMESTAMP DEFAULT NOW() NOT NULL,
+    expire_dtimes TIMESTAMP,
+    signature VARCHAR,
+    hash VARCHAR,
+    accepted_claims VARCHAR,
+    permitted_scopes VARCHAR,
+    PRIMARY KEY (id)
+);
```

Comment:
Lets have only the 3 tables that is relavent to keymanager and avoid the eSignet tables.

Reply:
removed other tables